### ISSUE TITLE
Fix today marker on grid layout

### ISSUE THREAD

**Issue by AlexxNica** [pln-planning-tools/MapLight#34]
Today marker is not currently working for the grid layout, and we have to find the best approach to render it.

**Comment by AlexxNica** [comment]
@reidlw is this critical for the MVP release?

**Comment by reidlw** [comment]
Yes, I think so. What's in the image here is what it should look like always. This looks rather broken/poorly implemented to me otherwise, and it's really important we get a good "first impression"


<img width="648" alt="Screen Shot 2022-10-25 at 11 14 57 PM" src="https://user-images.githubusercontent.com/17789009/197892295-e91e2b2d-062a-4e96-995e-53744b76f336.png">

**Comment by juliaxbow** [comment]
OK by me, especially since the ETAs really are just estimates and not due dates. Also I think we can expect folks this time around to be able to figure out "today" :) 

+1 to toggle-ability, especially if we're expecting that people will screenshot roadmaps for presentation purposes
